engine/classes/ElggSite.php

```diff
@@ -221,7 +214,7 @@ public function setDisplayName($displayName) {
 	 * Adds an entity to the site.
 	 *
 	 * This adds a 'member_of_site' relationship between between the entity and
-	 * the site. It does not change the site_guid of the entity.
+	 * the site.
```

Comment:
Just wipe these as well. No point in keeping them around, as all entities belong to a single site